### DISCUSSION TITLE
Looking to add custom information to each report produced.

### DISCUSSION THREAD

**Discussion by Kulezic101** [ortoniKC/ortoni-report#46] [Ideas]
Hi,
We would like to record the version information of the target environment so that it is easily referenced while viewing the report.
In other words, we would like to know what version was tested by the software in order to produce the given report.

(Perhaps there is already a way of doing this, but I am unable to find it.)

Thanks,
Igor

**Comment by ortoniKC** [comment]
Hi, you can append the target name to the file name or use the test type name.

**Reply by Kulezic101** [reply]
Thanks,   we have a number of components to the whole each with a version number.  We would like to see all the versions.

**Comment by ortoniKC** [comment]
Can you please scribble on the screenshot and post here, so I can give you
custom fields accordingly.

**Comment by Kulezic101** [comment]
<img width="215" alt="image" src="https://github.com/user-attachments/assets/e99dbb25-d091-429c-8722-16357d4ea6a4">

Where App[1-6] Version would be a field name of application.

**Comment by ortoniKC** [comment]
![Screenshot_20250313_013647.jpg](https://github.com/user-attachments/assets/ec41af9c-2ed1-4db9-911d-fc9d6c0965c7)

Hope this helps

**Comment by ortoniKC** [comment]
Added in V3.0.0
